src/ixdat/plotters/ms_plotter.py

```diff
@@ -165,7 +165,7 @@ def plot_vs(
         unit=None,
         logplot=True,
         legend=True,
-        **kwargs,
+        **plot_kwargs,
```

Comment:
This also looks like a change I have seen before :|

Reply:
Or maybe it is just in a new plotter

Reply:
There's so many of these sneaky little kwargs that escaped a refactor.